lib/module.js

```diff
@@ -12,3 +12,5 @@ export default /* async */ function (moduleOptions) {
 }
 
 const DEFAULTS = {}
+
+module.exports.meta = require('../package.json')
```

Comment:
```suggestion
;(nuxtModule as any).meta = require('../package.json')
```

Seems to be required as `esbuild` otherwise puts the `module.exports` and `module.exports.meta` in the wrong order